marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/ModelControllerTest.java

```diff
@@ -60,26 +69,114 @@ private void updateModelInfo() {
             throw new RuntimeException(e);
         }
 
+        assertSearchOptionsDeployment();
+        assertPIIFilesDeployment();
+        assertIndexDeployment();
+
         assertEquals("string", loadModel(getHubClient().getFinalClient()).get("definitions").get(MODEL_NAME).get("properties").get("someProperty").get("datatype").asText());
     }
 
     private JsonNode loadModel(DatabaseClient client) {
         return client.newJSONDocumentManager().read("/entities/" + MODEL_NAME + ".entity.json", new JacksonHandle()).get();
     }
+
+    private void assertSearchOptionsDeployment() {
+        DatabaseClient stagingDatabaseClient = getHubClient().getStagingClient();
+        DatabaseClient finalDatabaseClient = getHubClient().getFinalClient();
+        Map<String, DatabaseClient> clientMap = new HashMap<>();
+        clientMap.put("staging", stagingDatabaseClient);
+        clientMap.put("final", finalDatabaseClient);
+        clientMap.forEach((databaseKind, databaseClient) -> {
+            QueryOptionsManager queryOptionsManager = databaseClient.newServerConfigManager().newQueryOptionsManager();
+            assertTrue(queryOptionsManager.readOptionsAs("exp-" + databaseKind + "-entity-options", Format.XML, String.class).contains(MODEL_NAME));
+            assertTrue(queryOptionsManager.readOptionsAs(databaseKind + "-entity-options", Format.XML, String.class).contains(MODEL_NAME));
+        });
+    }
+
+    private void assertPIIFilesDeployment() {
+        runAsAdmin();
+        ManageClient manageClient = getHubClient().getManageClient();
+
+        try {
+            assertTrue(manageClient.getJson("/manage/v2/protected-paths").contains("someProperty"));
+
+            JsonNode queryRolesets = objectMapper.readTree(manageClient.getJson("/manage/v2/query-rolesets"));
+            assertTrue(queryRolesets.get("query-roleset-default-list").get("list-items").get("list-count").get("value").asInt() >= 1);
+        }
+        catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private void assertIndexDeployment() {
+        ManageClient manageClient = getHubClient().getManageClient();
+        Stream.of(getHubConfig().getDbName(DatabaseKind.STAGING), getHubConfig().getDbName(DatabaseKind.FINAL)).forEach(databaseKind -> {
+            String indexConfig = manageClient.getJson("/manage/v2/databases/" + databaseKind + "/properties");
+            assertTrue(indexConfig.contains("testRangeIndexForDHFPROD4704"));
+            assertTrue(indexConfig.contains("testPathIndexForDHFPROD4704"));
+            assertTrue(indexConfig.contains("someProperty"));
+            assertTrue(indexConfig.contains("someOtherProperty"));
+        });
+    }
+
+    private void loadUnrelatedIndexes() {
+        String indexConfig = "{\n" +
+            "   \"lang\":\"zxx\",\n" +
+            "   \"path-namespace\":[\n" +
+            "      {\n" +
+            "         \"prefix\":\"es\",\n" +
```

Comment:
Shouldn't this be a different namespace - isn't this the one that's generated too?

Reply:
Yep, forgot to change it while copying.
Updated in here b6777af